### PR TITLE
feat: add sms campaigns component

### PR DESCRIPTION
**Pull request by punwai**
## Description
- Add SMS campaigns component

## Related issues and discussion
#166 

## Screenshots, if any
smscampaigns

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by punwai** [comment]
Please tell me what you think about the tooltip using the titlecase, I am not sure whether this is the way that this should be done.

**Comment by punwai** [comment]
Thanks alot! @abhaychawla @Jov03

File: src/app/organization/sms-campaigns/sms-campaigns.component.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <ng-container matColumnDef="triggerType.value">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Trigger Type </th>
+        <td mat-cell *matCellDef="let smsCampaign"> {{ smsCampaign.triggerType.value }} </td>
+      </ng-container>
+      
+      <ng-container matColumnDef="campaignStatus.value">
+        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <td mat-cell *matCellDef="let smsCampaign">
+          <div [className]="smsCampaign.campaignStatus.code | statusLookup">
+            <fa-icon matTooltip="{{ smsCampaign.campaignStatus.value | titlecase }}" matTooltipPosition="right" icon="circle" size="lg"></fa-icon>
```
Comment (punwai) [inline src/app/organization/sms-campaigns/sms-campaigns.component.html]: Please tell me what you think about doing this for the tooltip. I have not seen any other components doing this yet so I am not too sure about it.

Comment (abhaychawla) [reply src/app/organization/sms-campaigns/sms-campaigns.component.html]: @punwai Good work!

**Pull request merged**